spec.html

```diff
@@ -3856,7 +3867,7 @@ <h1>ToInt32 ( _argument_ )</h1>
       <emu-alg>
         1. Let _number_ be ? ToNumber(_argument_).
         1. If _number_ is *NaN*, *+0*, *-0*, *+&infin;*, or *-&infin;*, return *+0*.
-        1. Let _int_ be the mathematical value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
+        1. Let _int_ be the Number value that is the same sign as _number_ and whose magnitude is floor(abs(_number_)).
```

Comment:
Why are these changes safe to make?

Reply:
Because the values involved are within the range where the calculation means the same thing for Numbers as well as mathematical values. Almost all of the specification is this way, and analogous changes are implicitly made everywhere else (based on the rewriting of the above section for mathematical operations).